tests/loaders/test_env_loader.py

```diff
@@ -102,5 +107,19 @@ def test_remap(monkeypatch):
             'val2': 'attr2',
         }
     ))
+    assert config.attr2 == 4    
     assert config.inner.attr1_name == 3
-    assert config.attr2 == 4
+    
+    
+def test_dotenv_loading(monkeypatch):
```

Comment:
I think it would also be ok to just create a sample .env-file (next to the sample yaml- and json-file) and select this file in the ConfZEnvSource with the correct path. This might test the indended use more closely than mocking os and io-stuff.

Reply:
Done.